config/locales/en.yml

```diff
@@ -78,9 +78,12 @@ en:
         research experience in this project was possible with support from the
         UNL Undergraduate Creative Activities and Research Experience Program.
         The UNL Center for Digital Research in the Humanities has contributed
-        considerable technical and financial resources to this project.</p>
-    proto: 
-      title: proto1
+        considerable technical and financial resources to this project.</p>    
+    proto:
+      title: proto
+      html: |-
+        <p>here goes the text. with grammar mistakes because there are no tildes in this computer.</p>    
```

Comment:
I will go in and change this text to read something along the lines of "this page is under construction."